src/content/release/compatibility-policy.md

```diff
@@ -43,12 +43,9 @@ break them overnight. This is independent of our compatibility policy
 which is exclusively based on whether submitted tests fail, as
 described above.
 
-Deprecated APIs are removed after a migration grace period. This grace
-period is one calendar year after being released on the stable channel,
-or after 4 stable releases, whichever is longer.
-
-When a deprecation does reach end of life, we follow the same procedures
-listed above for making breaking changes in removing the deprecated API.
+Deprecated APIs are not removed on a scheduled basis. If any are removed,
+we follow the same procedures listed above for making breaking changes
+in removing the deprecated API.
```

Comment:
**issue:** Cleaned up based on Google Style rules. Write in active voice per [GSG](https://developers.google.com/style/voice). (Also, some word list issues.)
Also, the 

```suggestion
The Flutter team doesn't remove deprecated APIs on a scheduled basis.
If the team removes a deprecated API,
it follows the same procedures as those for breaking changes.
```